homeassistant/components/thermostat/eq3btsmart.py

```diff
@@ -77,13 +77,13 @@ def device_state_attributes(self):
     def min_temp(self):
         """Return the minimum temperature."""
         return convert(self._thermostat.min_temp, TEMP_CELSIUS,
-                       self.unit_of_measurement)
+                       self.unit_of_measurement)[0]
```

Comment:
Damnit - missed it - this is one of the ones without unit tests that I mentioned in gitter.